src/System.CommandLine/Parsing/CommandResult.cs

```diff
@@ -26,12 +24,8 @@ public class CommandResult : SymbolResult
 
         public ICommand Command { get; }
 
-        public OptionResult? this[string alias] => OptionResult(alias);
-
-        public OptionResult? OptionResult(string alias)
-        {
-            return Children[alias] as OptionResult;
-        }
+        // FIX: (CommandResult) remove indexer
+        public OptionResult? this[string alias] => Children.GetByAlias(alias) as OptionResult;
```

Comment:
The issue was remove or rationalize the indexers. Is there a reason we are leaving this one in?

Reply:
I had forgotten to push the last commit.